app.rb

```diff
@@ -1,8 +1,52 @@
 require 'sinatra/base'
+require 'pg'
+require_relative './lib/user'
+require_relative './lib/peeps_manager'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
+  enable :sessions
+
+  get '/' do
+    erb(:sign_in)
+  end
+
+  post '/sign_in_conf' do
```

Comment:
nice work with sign in!

If you want to check that a user is signed in before all controller actions, look up 'sinatra controller hooks' and you'll find a way of executing some code before any controller action (and you can check for a user in the session there)